test/test_models_recognition.py

```diff
@@ -164,3 +164,23 @@ def test_compute_loss_sar(test_compute_target_sar):
     encoded_gts, seq_len = test_compute_target_sar
     loss = model.compute_loss(encoded_gts, model_output, seq_len)
     assert isinstance(loss, tf.Tensor)
+
+
+@pytest.fixture(scope="function")
+def test_compute_target_crnn():
+    list_gts = ['elephants', '1234', 'Rémouleur']
+    model = recognition.crnn_vgg16_bn()
+    encoded_gts, seq_len = model.compute_target(list_gts)
+    assert isinstance(encoded_gts, tf.Tensor)
+    assert isinstance(seq_len, tf.Tensor)
+    assert list(seq_len.numpy()) == [9, 4, 9]
+    return encoded_gts, seq_len
```

Comment:
We will be able to refactor this test with the SAR if we implement my above comment